### ISSUE TITLE
react/jsx-indent errors not reported on CircleCI

### ISSUE THREAD

**Issue by tleunen** [yannickcr/eslint-plugin-react#858]
For some reasons, CircleCI doesn't report any errors related to jsx-indent... When running it locally or in the editor reports the error.

Does anyone know why? And a way to fix it?

**Comment by lencioni** [comment]
Are you using the same version of eslint, eslint plugins, and eslint config
locally and in CI? Can you share your eslint config and CI output here?

**Comment by tleunen** [comment]
Yep. Same version. I even ssh on the container in circleci to check the file and I can see the indentation issue.

I can't share a public url because it's a private project but here the output on circleci:
```
> myproject@0.1.0 pretest /home/ubuntu/myproject
> npm run lint

> myproject@0.1.0 lint /home/ubuntu/myproject
> eslint --ext .jsx,.js src

The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint-plugin-import instead.

> myproject@0.1.0 test /home/ubuntu/myproject
> cross-env BABEL_ENV=cjs jest --coverage
```
So it didn't report anything before starting the tests...

The config is the one from airbnb (latest version), with these overrides:
```
"react/jsx-indent": [2, 4],
"react/jsx-indent-props": [2, 4],
"react/prop-types": [2, { "ignore": ["children"] }],
"react/style-prop-object": 0,
```

**Comment by tleunen** [comment]
Hmm... switched from Ubuntu 12.04 to 14.04 (not sure why this project was still on 12.04) and now the error is reported...

😕

**Comment by lencioni** [comment]
Weird! I'm glad you figured it out!
